app/models/book.rb

```diff
@@ -5,4 +5,16 @@ class Book < ApplicationRecord
 
   validates :name, presence: true
   validates :release, presence: true
+
+  def all__reviews_comment
+    reviews.pluck(:comment).join(', ')
+  end
+
+  def get_avg_star
+    return 'No review' unless reviews.present?
+
+    star_sum = reviews.reduce(0) { |prev, curr| prev + curr.star }
+
+    star_sum / reviews.length
```

Comment:
Not sure this work or not
```
reviews.average(:star)
```
ref: https://apidock.com/rails/ActiveRecord/Calculations/average